twentytwentyone-blocks/experimental-theme.json

```diff
@@ -136,6 +136,40 @@
 						"name": "Gigantic"
 					}
 				],
+				"fontWeights": [
+					{
+						"slug": "light",
+						"value": 300,
+						"name": "Light"
+					},
+					{
+						"slug": "normal",
+						"value": "normal",
+						"name": "Normal"
+					},
+					{
```

Comment:
Sorry, I noticed that a few things use a `500` weight too, so we should probably add that in: 

https://github.com/WordPress/twentytwentyone/blob/30379d158e67cd33d02ecd0b68970cfb4609b7c9/assets/sass/01-settings/global.scss#L123